packages/core/src/lib/implementation/execute-plugin.spec.ts

```diff
@@ -105,6 +105,16 @@ describe('executePlugin', () => {
 });
 
 describe('executePlugins', () => {
+  let errorSpy: SpyInstance;
+
+  beforeEach(() => {
+    errorSpy = vi.spyOn(console, 'error');
```

Comment:
Nice! we should replace also other places with this mock. 
ATM we use no mocking techniques in many place.

cc @Tlacenka

Reply:
It's a good idea, but I'd propose it to do outside of the scope of this task

Reply:
yes not here

Reply:
This will be resolved in #277